packages/@lwc/integration-karma/test/rendering/version-mismatch/index.spec.js

```diff
@@ -110,5 +106,34 @@ if (!process.env.COMPAT) {
                 );
             });
         });
+
+        describe('only warns once', () => {
```

Comment:
Related to the earlier comment: This test is a false negative. May be try registering a template with a different version or registering a component and verify that the message is logged only once.